src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -114,26 +112,31 @@ private void InitializeServices()
 
             _isHttpWorker = _serviceProvider.GetService<IOptions<HttpWorkerOptions>>()?.Value?.Description != null;
             _scriptOptions = _serviceProvider.GetService<IOptions<ScriptJobHostOptions>>();
-            _languageWorkerOptions = _serviceProvider.GetService<IOptionsMonitor<LanguageWorkerOptions>>();
 
             // Resetting the logger switches the logger scope to Script Host level,
             // also making the logs available to Application Insights
             _logger = _serviceProvider?.GetService<ILoggerFactory>().CreateLogger(LogCategories.Startup);
             _servicesReset = true;
         }
 
+        private IList<RpcWorkerConfig> GetFallbackWorkerConfig()
+        {
+            return _serviceProvider.GetService<IOptionsMonitor<LanguageWorkerOptions>>().CurrentValue.WorkerConfigs;
```

Comment:
maybe just add a comment about how we'll get the current jobhost config if one is not supplied directly?

Reply:
Agreed. Would be good to document the behavior this enables.

Reply:
done